.github/workflows/conventional_commits_check.yml

```diff
@@ -0,0 +1,18 @@
+name: Continous integration
+description: 'Github action to check conventional commits on OAT pull requests'
+
+on:
+  pull_request:
+    branches: [ develop ]
```

Comment:
Is it relevant to create releases on `develop` rather than `master` branch?

Reply:
release tool merges develop in master and then makes a release from master branch.